content/quickstart.md

```diff
@@ -345,3 +345,97 @@ and sends invoices to clients on the same schedule.
+  refactoring from others
+- Uses outdated or inappropriate tools, technologies, or methodologies
+- Avoids or gives up on challenges
+- Does not plan or dissect big assignments into smaller tasks
+
+#### Meets Expectations
+
+- Takes initiative on high priority work by chipping away at it
+- Delivers complete and high-quality work, and acts on feedback from others
+- Approaches challenges by spending time planning and dissecting out assignments
+
+#### Exceeds Expectations
+
+- Takes ownership of features and infrastructure
+- Leads internal review processes and delivers high-quality, actionable feedback
+  to others
+- Makes requests for input early in process to tackle challenging work
+- Takes initiative and leads others in planning and breaking down upcoming work
+
+### Communication
+
+#### Underperforms
+
+- Rarely communicates or does so in an untimely manner
+- Misses large parts of or entire meetings without notice
+- Does not document the breakdown of upcoming work or the status of in-progress
+  work, and does not reflect on their contributions, accomplishments, or lessons
+  from challenges (self-retro)
+- Does not ask for help or guidance when blocked, and does not participate or
+  learn from review and discussions
+- Shows disinterest or indifference when planning new work
+
+#### Meets Expectations
+
+- Communicates frequently and in a timely manner
+- Attends most meetings and notifies others in advance when they will be late or cannot attend
+- Posts a breakdown of their daily assignments documenting completed work, the
+  status of in-progress work, and upcoming tasks
+- Documents their contributions, accomplishments, and lessons from challenges
+  (self-retro)
+- Reaches out with questions when blocked and engages with review and
+  planning by asking follow-up questions
+
+#### Exceeds Expectations
+
+- Communicates with trust in advance and requests engagement from others
+- Leads meetings and discussions about internal and external projects
+- Follows up during and after reviews and discussions with questions that help
+  confirm understanding
+- Generates internal documents, blog posts, and other valuable content based on
+  their contributions (especially accomplishments and challenges)
+
+### Learning & Growth
+
+#### Underperforms
+
+- Does not participate or learn from review and discussions, and asks few or
+  no questions
+- Resists or rejects change and feedback, and does not adjust to the changing
+  needs and expectations of projects
+- Lacks initiative and collaboration skills, and does not support the team's
+  goals and objectives
+
+#### Meets Expectations
+
+- Engages with review and discussions by asking more questions
+- Reaches out with questions when blocked
+- Acts on feedback as an opportunity to learn and improve, and remains flexible and
+  adaptable to change
+
+#### Exceeds Expectations
+
+- Demonstrates curiosity and pursues challenges that help them and the team grow
```

Comment:
I think the rest of the quickstart was written in second person. The change didn't stand out to me until this use of `them`. Not sure it matters much.

Reply:
Switched to second person in 84dd72f9ebf644d93efa4e1b16a37a27571a5ae9